src/otx/core/model/detection.py

```diff
@@ -74,26 +74,33 @@ def forward_tiles(self, inputs: TileBatchDetDataEntity) -> DetBatchPredEntity:
         tile_attrs: list[list[dict[str, int | str]]] = []
         merger = DetectionTileMerge(
             inputs.imgs_info,
+            self.tile_config.tile_size[0],
```

Comment:
`self.tile_config.tile_size` seems a tuple of two integers but I guess that you assume that they are square (`tile_size[0] == tile_size[1]`. How do you guarantee it?

Reply:
Thank you for pointing that out, I updated the algo to cover the use case when the tile is not square.